.nvidia-ci.yml

```diff
@@ -419,3 +419,72 @@ release:ngc-rhel9.3:
     - .dist-rhel9
   variables:
     OUT_DIST: "rhel9.3"
+
+
+# Define the external image signing steps for NGC
+# Download the ngc cli binary for use in the sign steps
+.ngccli-setup:
+  before_script:
+    - apt-get update && apt-get install -y curl unzip jq bash make
+    - export REGCTL_VERSION=v0.4.7
+    - mkdir -p bin
+    - curl -sSLo bin/regctl https://github.com/regclient/regclient/releases/download/${REGCTL_VERSION}/regctl-linux-amd64
+    - chmod a+x bin/regctl
```

Comment:
Why are we installing `regctl` here too? Looking at the `.sign:ngc` job this seems more complex than the other case and I would have to check to see if there is a way to simplify this somewhat.

Reply:
we are using precompiled.sh to import kernel version and dist tag . 
./scripts/precompiled.sh version;
 source kernel_version.txt
 export IMAGE_TAG=${DRIVER_BRANCH}-${KERNEL_VERSION}-${DIST##*_}

precompiled.sh :- 
function sourceVersions(){    --- use regctl 
---- 
if command -v regctl; then    


The KERNEL_VERSION and DIST for pre-compiled images differ from those of regular container images. 
We can simplify the process if we obtain them through other means.